src/main.js

```diff
@@ -1,10 +1,12 @@
 'use strict';
 
 const INSTAGRAM_HOSTNAME = 'www.instagram.com';
+const UNFOLLOWERS_PER_PAGE = 75;
```

Comment:
This is the type of thing that in the future should be made configurable for the user.
Maybe I'll get around to it in a future update :)

Reply:
We should make a setting icon with a dropdown menu.